requirements.txt

```diff
@@ -4,3 +4,4 @@ qiskit-terra>=0.19.0
 qiskit-ibmq-provider>=0.16.0
 matplotlib>=3.4
 uncertainties
+pandas
```

Comment:
Better verify we are not using relatively new pandas features if we don't specify a version.

Reply:
Do you have an idea how to do it?

Reply:
Not really; I'd simply look what pandas version was in use two years ago, install it explicitly, verify everything works with it and add to the requirements.

Reply:
Done in f50063ae3fdf75b721da8fc5338aa1f6ee464d8b